inst/app/R/fct_dea-boostrap.R

```diff
@@ -0,0 +1,94 @@
+bw_rule <- function(delta, rule = 'cv', adjust = TRUE, type = NULL) {
+  bw_rule <- 3
+  if (is.character(rule)) {
+    bw_rule <- switch(rule,
+                      'silverman' = 1,
+                      'scott' = 2,
+                      'cv' = 3,
+                      'sw98' = 4,
+                      3
+    )
+    if (is.null(bw_rule)) {
+      cli::cli_warn('Unknown bandwidth rule. Using default value instead')
+    }
+  } else if (is.numeric(rule)) {
+    if (rule %in% 1:3) {
+      bw_rule <- rule
+    } else {
+      cli::cli_warn('Unknown bandwidth rule. Using default value instead')
+    }
+  } else {
+    cli::cli_warn('Unknown bandwidth rule. Using default value instead')
+  }
+  # Values must be in range 1, Inf. Take inverse if values are in range 0, 1
+  if (min(delta) < 1) {
+    delta <- 1/delta
+  }
+  # Values == 1 are artifacts of DEA method and should be removed for h
+  delta_m <- delta[delta > 1]
```

Comment:
Comment say values == 1, while code says values > 1 ?

Reply:
Values == 1 should be removed, as the code does 😉 (at this point, all values are in the range [1, ∞])

Reply:
Got it. What about using `!=`?

Reply:
This open us up further for floating point errors. It might be better to implement something along the lines of `delta >  1 + 1e-9` (this is done in Benchmarking with 1e-6)

Reply:
Good point. Using something like `delta >  1 + 1e-9 might be the best option.